osu.Game/Overlays/Profile/Sections/ProfileItemContainer.cs

```diff
@@ -16,12 +16,19 @@ public class ProfileItemContainer : Container
 
         protected override Container<Drawable> Content => content;
 
-        private Color4 idleColour;
-        private Color4 hoverColour;
-
         private readonly Box background;
         private readonly Container content;
 
+        private Color4 idleColour;
+
+        public Color4 IdleColour
+        {
+            get => idleColour;
+            set => idleColour = background.Colour = value;
```

Comment:
This compound assignment is sort of wrong compared to the previous implementation as the setter will overwrite hover colour if called while the item is hovered.

Reply:
Solved by checking for the container being hovered (and adding similar check to `HoverColour` as well) - hopefully that's enough.